open_spiel/games/checkers.h

```diff
@@ -0,0 +1,134 @@
+// Copyright 2019 DeepMind Technologies Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef OPEN_SPIEL_GAMES_CHECKERS_H_
+#define OPEN_SPIEL_GAMES_CHECKERS_H_
+
+// Implementation of the board game Checkers.
+// https://en.wikipedia.org/wiki/Checkers
+//
```

Comment:
I noticed your implementation has rows and columns as paramters. We usually list them all here (see headers of other games)